ML-for-S2S/ML-for-Derecho/torch_s2s_dataset_gust.py

```diff
@@ -0,0 +1,369 @@
+                f'{self.cesm_dir}cm_{self.variable_}_'+dt_string+'.nc') # cesm2 list
+            self.list_of_era5.append(
+                f'{self.era5_dir}e5_{self.variable_}_'+dt_string+'.nc') # era5 list
+            
+            
+    def zscore_values(self, mnv, stdv):
+        """
+        compute zscore values
+        """
+        # help with variable names inside files
+        if self.variable_ == 'tas2m':
+            var = 'tas_2m'
+        if self.variable_ == 'prsfc':
+            var = 'pr_sfc'
+        if self.variable_ == 'prsfc_anom' or self.variable_ == 'tas2m_anom':
+            var = 'anom'
+        
+        # if mean and standard deviation are NOT provided do this (only era5)
+        if mnv == None or stdv == None:
+        
+            self.mean_val = xr.open_mfdataset(
+                self.list_of_era5, concat_dim='sample', combine='nested')[var].mean(
+                skipna=True).values
+
+            self.std_val = xr.open_mfdataset(
+                self.list_of_era5, concat_dim='sample', combine='nested')[var].std(
+                skipna=True).values
+            
+        # if mean and standard deviation ARE provided do this
+        if mnv != None and stdv != None:
+            
+            self.mean_val = mnv
+            self.std_val = stdv
+        
+
+    def minmax_values(self, minv, maxv):
+        """
+        compute minmax values
+        """
+        # help with variable names inside files
+        if self.variable_ == 'tas2m':
+            var = 'tas_2m'
+        if self.variable_ == 'prsfc':
+            var = 'pr_sfc'
+        if self.variable_ == 'prsfc_anom' or self.variable_ == 'tas2m_anom':
+            var = 'anom'
+        
+        # if min and max are NOT provided do this (only era5)
+        if minv == None or maxv == None:
+            
+            self.max_val = xr.open_mfdataset(
+                self.list_of_era5, concat_dim='sample', combine='nested')[var].max(
+                skipna=True).values
+        
+            self.min_val = xr.open_mfdataset(
+                self.list_of_era5, concat_dim='sample', combine='nested')[var].min(
+                skipna=True).values
+            
+        # if min and max ARE provided do this
+        if minv != None and maxv != None:
+            
+            self.max_val = maxv
+            self.min_val = minv
+        
+        
+    def create_files(self, indx):
+        """
+        create input and label data using file list and indx from sampling
+        """
+        # help with variable names inside files
+        if self.variable_ == 'tas2m':
+            var = 'tas_2m'
+        if self.variable_ == 'prsfc':
+            var = 'pr_sfc'
+        if self.variable_ == 'prsfc_anom' or self.variable_ == 'tas2m_anom':
+            var = 'anom'
+            
+        # coordinates (terrain and lat/lon features)
+        self.coord_data = xr.open_dataset(
+            '/glade/gust/scratch/molina/ml_coords.nc').expand_dims('sample')
```

Comment:
instead of hard coding the directory string, can we do this instead (I am doing this on gust now and it works) 

`self.homedir+'/ml_coords.nc'`

Reply:
Otherwise -- everything else looks great!

Reply:
Sounds good! I'm making this change now and will push up.

Reply:
looks great!